core/src/main/java/nl/nn/adapterframework/senders/XsltSender.java

```diff
@@ -348,6 +263,59 @@ private ContentHandler createHandlerOud(String correlationID, String input, IPip
 			throw new StreamingException(getLogPrefix()+"Exception on creating transformerHandler chain", e);
 		} 
 	}
+	
```

Comment:
Kan dit weg?

Reply:
Ik kreeg commentaar van een collega dat ik allemaal uitgecommentarieerde en/of anderszins ongebruikte code achterliet...